package.json

```diff
@@ -59,6 +58,7 @@
     "@typescript-eslint/parser": "4.28.0",
     "babel-eslint": "10.1.0",
     "eslint": "7.29.0",
+    "eslint-config-next": "^11.0.1",
```

Comment:
Can you pin this dependency as well? 😉 

We should use `yarn add` with the `--exact` option to pin the dependency in the `package.json` and `yarn.lock` file when we're installing new stuff into our repo. 😄

Reply:
Damn it! I'll do that and make sure all versions are fixed with `.yarnrc` file.